sql/mysqld.cc

```diff
@@ -4421,6 +4424,83 @@ a file name for --log-bin-index option", opt_binlog_index_name);
     unireg_abort(MYSQLD_ABORT_EXIT);
   }
 
+  if (opt_libcoredumper)
+  {
+#ifndef HAVE_LIBCOREDUMPER
+    sql_print_warning(
+        "This version of MySQL has not been compiled with "
+        "libcoredumper support, ignoring --coredumper argument");
+#else
+    if (opt_corefile)
+    {
+      sql_print_warning(
+          "Started with --core-file and --coredumper. "
+          "--coredumper will take precedence.");
+    }
+    if (opt_libcoredumper_path != NULL)
+    {
+      /* validate path */
+      if (!is_valid_log_name(opt_libcoredumper_path,
+                             strlen(opt_libcoredumper_path)))
+      {  //filename contain .cnf or .ini on it
```

Comment:
I'm not sure if I understand this condition - why sepcifically ini file?

Reply:
This is just to make sure we are not using something that can be interpreted as config file. is_valid_log_name function test it for linux(.cnf) and windows(.ini)